src/utils/checkpid.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+	"github.com/facebookgo/pidfile"
+)
+
+func CheckPid(pidfileName string) error {
+	pidfile.SetPidfilePath(pidfileName)
+	pid, err := pidfile.Read()
```

Comment:
For this, you should not introduce additional dependence:
```
	d, err := ioutil.ReadFile(*pidfile)
	if err != nil {
		return 0, err
	}

	pid, err := strconv.Atoi(string(bytes.TrimSpace(d)))
	if err != nil {
		return 0, fmt.Errorf("error parsing pid from %s: %s", *pidfile, err)
	}
```
I suggest not to use `github.com/facebookgo/pidfile`

Reply:
IMHO I do not like this approach.
Of the entire library, only one function is used. The function is quite small.
Ignore it `pidfile.SetPidfilePath`.

Reply:
github.com/facebookgo/pidfile provides 5 functions: IsNotConfigured, GetPidfilePath, SetPidfilePath, Write, Read. 
This pull-request uses 3 of them:  SetPidfilePath, Write, Read; which covers more that 85% of source code of github.com/facebookgo/pidfile

Reply:
Cool. You calculated the use. 
My arguments about the volume were not entirely correct.
I still see no point in this dependence. 
Approved it. Do as you see fit.